core/src/main/java/org/wso2/testgrid/core/TestPlanExecutor.java

```diff
@@ -52,7 +56,9 @@
  */
 public class TestPlanExecutor {
 
+    public static final int LINE_LENGTH = 72;
     private static final Logger logger = LoggerFactory.getLogger(TestPlanExecutor.class);
+    private static final int MAX_NAME_LENGTH = 52;
```

Comment:
Any reasons to use these numbers for lengths? ( TMK :) )

Reply:
Only requirement was to keep the scenario pass/fail status lines within the LINE_LENGTH (72) character limit. 20 characters were reserved to print things like '[INFO]' and 'SUCCESS/FAIL'... :)